container/pantheon-karaf.yml

```diff
@@ -0,0 +1,91 @@
+# Generation of Kubernetes YAML is still under development!
+#
+# Save the output of this file and use kubectl create -f to import
+# it into Kubernetes.
+#
+# Created with podman-1.9.2
+apiVersion: v1
+kind: Pod
+metadata:
+  creationTimestamp: "2020-06-02T11:45:11Z"
+  labels:
+    app: pantheon-karaf
+  name: pantheon-karaf
+spec:
+  containers:
+  - command:
+    - /opt/pantheon-karaf-dist/bin/karaf
+    - run
+    env:
+    - name: PATH
+      value: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+    - name: TERM
+      value: xterm
+    - name: HOSTNAME
+      value: pantheon-karaf
+    - name: MONGO_DB
+      value: sling
+    - name: REPOSITORY
+      value: sling/repository
+    - name: MONGO_DB_REPLICA
+      value: mongodb://localhost:27017
+    - name: container
+      value: oci
+    image: localhost/pantheon-karaf-app:latest
+    name: pantheon-karaf-app
+    ports:
+    - containerPort: 8181
+      hostPort: 8181
+      protocol: TCP
+    - containerPort: 55555
+      hostPort: 55555
+      protocol: TCP
+    - containerPort: 1099
+      hostPort: 1099
+      protocol: TCP
+    resources: {}
+    securityContext:
+      allowPrivilegeEscalation: true
+      capabilities: {}
+      privileged: false
+      readOnlyRootFilesystem: false
+      seLinuxOptions: {}
+    tty: true
+    workingDir: /tmp/archive
+  - command:
```

Comment:
mongoDB as container, is it for testing purpose only ?

Reply:
This file was generated using podman generate. I am not sure is this required. I have asked Lisa about it. I will remove this and regenerate if required. This is useful only if you are using kubectl, AFAIK. And this seems only for local.